### PR TITLE
Master

### PR DESCRIPTION
**Pull request by M-Khairy-B**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,967 @@
...
+            href="https://fonts.googleapis.com/css2?family=Salsa&display=swap"
+            rel="stylesheet"
+        />
+        <link
+            href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&family=Salsa&display=swap"
+            rel="stylesheet"
+        />
+        <link href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100..900;1,100..900&display=swap" rel="stylesheet">
+        <link
+            href="https://fonts.googleapis.com/css2?family=Josefin+Sans:ital,wght@0,100..700;1,100..700&family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&family=Montserrat:ital,wght@0,100..900;1,100..900&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&family=Salsa&display=swap"
```
Comment (mukhtar-ali) [inline index.html]: One import is enough.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,967 @@
...
+                </div>
+            </nav>
+        </header>
+
+        <section class="header">
+            <div class="container">
+                <div class="row">
+                    <div class="col-1 col-lg-1 header-space">
+                        <div class="line-left"></div>
+                    </div>
```
Comment (mukhtar-ali) [inline index.html:100]: Why did you make the space like this? You should use flex properties to align the items center instead.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,967 @@
...
+                    <div class="col-11 col-lg-5 header-space">
+                        <h1>
+                            PS4 V2 <br />
+                            DualShock 4
+                        </h1>
+                        <p class="m-auto ms-lg-auto">
+                            wireless controller for playStation 4
+                            (compatibilityCentric)
```
Comment (mukhtar-ali) [inline index.html:108]: compatibilityCentric style not like the design

File: index.html [hunk 4]
```diff
@@ -0,0 +1,967 @@
...
+                <div class="row multiple-items">
+                    <div class="col-md-12">
+                        <a href=""></a>
+                        <div class="layer product-details">
+                            <img
+                                class="laptop"
+                                src="img/kisspng-intel-monster-tulpar-t5-v18-monster-abra-a5-v13-mo-pin-v14-on-pinterest-5be2d46da89577 3.png"
+                                alt=""
+                            />
+                            <h4>Laptops</h4>
```
Comment (mukhtar-ali) [inline index.html:232]: Not like the design

File: index.html [hunk 5]
```diff
@@ -0,0 +1,967 @@
...
+                </div>
+                <div class="row multiple-items">
+                    <div class="col-md-12">
+                        <a href=""></a>
+                        <div class="layer product-details">
+                            <img
+                                class="laptop"
+                                src="img/kisspng-intel-monster-tulpar-t5-v18-monster-abra-a5-v13-mo-pin-v14-on-pinterest-5be2d46da89577 3.png"
+                                alt=""
+                            />
```
Comment (mukhtar-ali) [inline index.html:231]: Not like the design

File: index.html [hunk 6]
```diff
@@ -0,0 +1,967 @@
...
+                    </div>
+                </div>
+            </div>
+        </section>
+        <section class="leatest-Products">
+            <div class="container text-center">
+                <div class="">
+                    <h2>Leatest Products</h2>
+                </div>
+                <ul
```
Comment (mukhtar-ali) [inline index.html:301]: Space between sections are not like the design

File: index.html [hunk 7]
```diff
@@ -0,0 +1,967 @@
...
+            </div>
+        </section>
+        <section class="products">
+            <div class="container text-center">
+                <div class="h-section">
+                    <h2>Featured Products</h2>
+                </div>
+                <div class="row multiple-items">
+                    <div class="col-md-12">
+                        <a href=""></a>
```
Comment (mukhtar-ali) [inline index.html]: ??

File: index.html [hunk 8]
```diff
@@ -0,0 +1,967 @@
...
+                                alt=""
+                            />
+                            <h4>Laptops</h4>
+                        </div>
+                    </div>
+                    <div class="col-md-12">
+                        <div class="layer product-details">
+                            <img class="headest" src="img/Headset.png" alt="" />
+                            <h4>HeadPhones</h4>
+                        </div>
```
Comment (mukhtar-ali) [inline index.html:239]: Missing anchor tag

Comment (mukhtar-ali) [inline index.html:239]: Missing backdrop shadow

Comment (mukhtar-ali) [inline index.html:239]: Missing backdrop shadow

File: index.html [hunk 9]
```diff
@@ -0,0 +1,967 @@
...
+                            </p>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </section>
+        <section class="products">
+            <div class="container text-center">
+                <div class="trending">
+                    <h2>Trending Products</h2>
```
Comment (mukhtar-ali) [inline index.html:655]: Space between sections are not like the design

**Review by mukhtar-ali — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (M-Khairy-B) [reply index.html]: Done

File: index.html (hunk 5 above)
Comment (M-Khairy-B) [reply index.html:231]: because of he size of the pictures

File: index.html (hunk 7 above)
Comment (M-Khairy-B) [reply index.html]: Done

File: index.html (hunk 8 above)
Comment (M-Khairy-B) [reply index.html:239]: Done

Comment (M-Khairy-B) [reply index.html:239]: Done

Comment (M-Khairy-B) [reply index.html:239]: Done

File: index.html (hunk 9 above)
Comment (M-Khairy-B) [reply index.html:655]: Done

File: index.html (hunk 6 above)
Comment (M-Khairy-B) [reply index.html:301]: Done

File: index.html [hunk 10]
```diff
@@ -0,0 +1,967 @@
...
+                            <img
+                                class=""
+                                src="img/oculus_quest_2_64gb_gafas_realidad_virtual_01_l-removebg-preview 1.png"
+                                alt=""
+                            />
+                            <h4>Wireless Headset</h4>
+                            <span>₹5999 </span>
+                        </div>
+                    </div>
+                    <div class="col-md-12">
```
Comment (mukhtar-ali) [inline index.html:669]: Not like the design

File: index.html [hunk 11]
```diff
@@ -0,0 +1,967 @@
...
+                                <h4 class="hover">Top esssential Trends in 2022</h4>
+                                <p>
+                                    NO STRINGS WITH THESE WIRELESS
+                                    <br />HEADPHONE
+                                </p>
+                                <span class="hover"> Read More</span>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="col-12 col-md-6 col-lg-4">
```
Comment (mukhtar-ali) [inline index.html:768]: Space between each card not like the design

**Review by mukhtar-ali — CHANGES_REQUESTED** [review]

**Pull request closed without merging**